src/me/components/CloudWidgets.tsx

```diff
@@ -46,26 +45,23 @@ export const CloudWidgets: FC = () => {
             body={
               <>
                 <p>
```

Comment:
We're also intentionally removing this info re: cloud dedicated, right? (since it's a past event?)

Reply:
👍